src/components/InngestFunction.ts

```diff
@@ -44,9 +44,9 @@ export class InngestFunction<Events extends Record<string, EventPayload>> {
   /**
    * The generated or given ID for this function.
    */
-  public get id() {
+  public id(prefix?: string) {
```

Comment:
I see what you mean - bit weird that `id` now always needs to be given a prefix and can no longer be a getter, but it makes sense as to why.